src/firmware/boards/robot_stm32h7/ethernetif.c

```diff
@@ -1,3 +1,7 @@
+// override default version of stm32h7xx_hal_eth.c with version from
+// https://community.st.com/s/question/0D50X0000C6eNNSSQ2/bug-fixes-stm32h7-ethernet
```

Comment:
:vomiting_face:

Reply:
so we just ignore the cube settings for now huh?

Reply:
yeah :( its a mess

they seem to be actively fixing bugs so hopefully this will go away soon :tm: